apps/auction2/client.js

```diff
@@ -70,12 +74,17 @@ const params = new Params(defaultParams, {
 })
 const filter = new Filter({ params: params })
 
-// Main Artworks view
+// REDUX
+const store = createStore(auctions)
+render(
+  <Provider store={store}>
+    <Works />
+  </Provider>,
+  document.getElementById('cf-artworks')
+);
+
 filter.artworks.on('reset', () => {
-  render(
-    <AuctionGrid artworks={filter.artworks.models} />,
-    document.getElementById('cf-artworks')
-  );
+  store.dispatch(actions.updateArtworks(filter.artworks.models))
```

Comment:
👍  this is typically how migrations between Backbone and React (via Flux) happen; event hooks at the root that then trigger actions. This makes it clear where the handshake takes place once Backbone is ready to be removed completely.